CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -492,16 +492,12 @@ public function parseActionSchedule($values) {
 
     if ($absoluteDate = CRM_Utils_Array::value('absolute_date', $params)) {
       $params['absolute_date'] = CRM_Utils_Date::processDate($absoluteDate);
-      $params['is_repeat'] = 0;
-      foreach ($moreKeys as $mkey) {
-        $params[$mkey] = 'null';
-      }
     }
     else {
       $params['absolute_date'] = 'null';
-      foreach ($moreKeys as $mkey) {
-        $params[$mkey] = CRM_Utils_Array::value($mkey, $values);
-      }
+    }
+    foreach ($moreKeys as $mkey) {
+      $params[$mkey] = CRM_Utils_Array::value($mkey, $values);
```

Comment:
@monishdeb Seems this change provides and extra ability to fill the values for absolute as well as relative dates. I think `$moreKeys` looping here should be containing only repetition field values.